city_scrapers/spiders/pitt_housing.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+import re
+import scrapy
+from datetime import date
+from html.parser import HTMLParser
+
+from dateutil.parser import parse as dtparse
+
+from city_scrapers.spider import Spider
+from city_scrapers.constants import *
+
+
+
+class PittHousingSpider(Spider):
+    name = 'pitt_housing'
+    agency_name = 'Housing Authority of the City of Pittsburgh'
+    timezone = 'America/New_York'
+    allowed_domains = ['www.hacp.org']
+    start_urls = ['http://www.hacp.org/public-information/board-info']
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` a dict that follows the Event Schema
+        <https://city-bureau.github.io/city-scrapers/06_event_schema.html>.
+
+        Change the `_parse_id`, `_parse_name`, etc methods to fit your scraping
+        needs.
+        """
+
+        for item in self._create_meetings(response): 
+
+            data = {
+                '_type': 'event',
+                'name': 'Housing Authority of the City of Pittsburgh',
```

Comment:
This isn't currently documented, but we want to avoid using the agency name as the meeting name, so in this case `Board of Commissioners` would work as the name, and you can leave the description blank since they don't provide a unique one for each meeting